multistate.cabal

```
@@ -68,7 +68,7 @@ library {
   other-modules:
     Control.Monad.Trans.MultiGST.Common
   build-depends:
-    base         >= 4.11  && <4.15,
+    base         >= 4.11  && <4.16,
```

I guess the package will be buildable with 4.16 as well
```suggestion
    base         >= 4.11  && <4.17,
```

---

No it is not, several errors like

```
[ 7 of 22] Compiling Control.Monad.Trans.MultiWriter.Strict ( src\Control\Monad\Trans\MultiWriter\Strict.hs, dist\build\Control\Monad\Trans\MultiWriter\Strict.o )

src\Control\Monad\Trans\MultiWriter\Strict.hs:123:3: warning: [-Wnoncanonical-monad-instances]
    Noncanonical ‘return’ definition detected
    in the instance declaration for ‘Monad (MultiWriterT x m)’.
    ‘return’ will eventually be removed in favour of ‘pure’
    Either remove definition for ‘return’ (recommended) or define as ‘return = pure’
    See also: https://gitlab.haskell.org/ghc/ghc/-/wikis/proposal/monad-of-no-return
    |
123 |   return = MultiWriterT . return
    |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```